formulas/dnsmasq/files/protect-dns-resolver-config

```diff
@@ -0,0 +1,19 @@
+eval "$(
+    funcBody=$(declare -f make_resolv_conf)
+    echo "${funcBody/make_resolv_conf/make_resolv_conf_real}"
+);"
+
+make_resolv_conf() {
+    resolv_conf=$(readlink -f "/etc/resolv.conf" 2>/dev/null) || resolv_conf="/etc/resolv.conf"
+    cp -p $resolv_conf /etc/resolv.conf.tmp || return 1
```

Comment:
Quote your variables.

You're still making the temp file in /etc instead of "${resolv_conf}.tmp"

Variable name is not declared `local`.

Variable name is not camel case.